README.md

```diff
@@ -317,6 +318,25 @@ Component structure:
 
 [↑ back to Table of Contents](#table-of-contents)
 
+## Component event names
+
+Vue.js provides events scoped to the component it emits and they are named as the developers desires. `~Make this paragraph better`
+
+### Why?
+
+* Developers are free to use native likes event names and it can cause confusion down the line;
+* The freedom of naming events can lead to a DOM templates incompatibility;
+* `~Make this paragraph better`
+
+### How?
+
+* Event names should be kebab-cased;
+* Event names should have a prefix strongly related to the name of the component in use (prefer `drive-upload-success` over `upload-succeeded` event name). This allows you to drop in multiple component in the page without event namespacing clashing ([source](https://github.com/GoogleWebComponents/style-guide#events));
+* A unique event name should be fired for unique actions in your component that will be of interest to the outside world, like dropzone-upload-success, dropzone-upload-error, todo-item-remove, todo-item-select, etc. ([source](https://github.com/GoogleWebComponents/style-guide#events));
```

Comment:
I would remove or adapt this part to avoid suggesting prefixed names. Is totally fine to use no prefixed ones as it was discussed.